docs/src/tutorials/getting_started/transitioning_from_matlab.jl

```diff
@@ -0,0 +1,389 @@
+# [`primal_status`](@ref), and [`raw_status`](@ref), See [Solutions](@ref jump_solutions)
+# for more details on how to query and interpret solution statuses.
+
+# ## Extracting variables
+
+# Like YALMIP, but unlike CVX, with JuMP you need to explicitly ask for the value
+# of your variables after optimization is done, with the function call `value(x)`
+# to obtain the value of variable `x`.
+
+value.(m[1][1, 1])
+
+# A subtlety is that, unlike YALMIP, the function `value` is only defined for
+# scalars. For vectors and matrices you need to use Julia broadcasting:
+# `value.(v)`.
+
+value.(m[1])
+
+# There is also a specialized function for extracting the value of the objective,
+# `objective_value(model)`, which is useful if your objective doesn't have a
+# convenient expression.
+
+objective_value(model)
+
+# ## Dual variables
+
+# Like YALMIP and CVX, JuMP allows you to recover the dual variables. In order
+# to do that, the simplest method is to name the constraint you're interested in,
+# for example, `@constraint(model, bob, sum(v) == 1)` and then, after the
+# optimzation is done, call `dual(bob)`. See [Duality](@ref) for more details.
+
+# ## Reformulating problems
+
+# Perhaps the biggest difference between JuMP and YALMIP and CVX is how far the
+# package is willing to go in reformulating the problems you give to it.
+
+# CVX is happy to reformulate anything it can, even using approximations if your
+# solver cannot handle the problem.
+
+# YALMIP will only do exact reformulations, but is still fairly adventurous,
+# for example, being willing to reformulate a nonlinear objective in terms of
+# conic constraints.
+
+# JuMP does no such thing: it only reformulates objectives into objectives, and
+# constraints into constraints, and is fairly conservative at that. As a result,
+# you might need to do some reformulations manually, for which a good guide is
+# the [Tips and tricks](@ref conic_tips_and_tricks) tutorial.
+
+# ## Vectorization
+
+# In MATLAB, it is absolutely essential to "vectorize" your code to obtain
+# acceptable performance. This is because MATLAB is a slow interpreted
+# language, which sends your commands to fast libraries. When you "vectorize"
+# your code you are minimizing the MATLAB part of the work and sending it to the
+# fast libraries instead.
+
+# There's no such duality with Julia.
+
+# Everything you write and most libraries you use will compile down to LLVM, so
+# "vectorization" has no effect.
+
+# For example, if you are writing a linear program in MATLAB and instead of the
+# usual `constraints = [v >= 0]` you write:
+# ```matlab
+# for i = 1:n
+#    constraints = [constraints, v(i) >= 0];
+# end
+# ```
+# performance will be poor.
+
+# With Julia, on the other hand, there is hardly any difference between
+# ```julia
+# @constraint(model, v >= 0)
+# ```
+#  and
+# ```julia
+# for i in 1:n
+#     @constraint(model, v[i] >= 0)
+# end
+# ```
+
```

Comment:
```suggestion

# ## Primal versus dual form

# When you translate some optimization problem from YALMIP or CVX to JuMP you 
# might be surprised to see it get much faster or much slower, even if you're 
# using exactly the same solver. The most likely reason is that YALMIP will
# always interpret the problem as the dual form, whereas CVX and JuMP will try to
# interpret the problem in the form most appropriate to the solver. If the 
# problem is more naturally formulated in the primal form it is likely that
# YALMIP's performance will suffer, or if JuMP gets it wrong its performance will
# suffer. It might be worth trying both primal and dual forms if you're having
# trouble, which can be done automatically with the package [Dualization](https://jump.dev/JuMP.jl/stable/packages/Dualization/#Dualization.jl).
#
# For an in-depth explanation of this issue see the [dualization](https://jump.dev/JuMP.jl/stable/tutorials/conic/dualization/) tutorial.
```